scss/tokens/_raw.scss

```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
+// Box-shadow
+$ouds-elevation-x-0:           0 !default;
+
+$ouds-elevation-y-0:           0 !default;
+$ouds-elevation-y-100:         1px !default;
+$ouds-elevation-y-200:         2px !default;
+$ouds-elevation-y-300:         4px !default;
+// $ouds-elevation-y-400:         8px !default;
+$ouds-elevation-y-500:         12px !default;
+// $ouds-elevation-y-600:         20px !default;
+
+$ouds-elevation-blur-0:        0 !default;
+// $ouds-elevation-blur-100:      1px !default;
+$ouds-elevation-blur-200:      2px !default;
+$ouds-elevation-blur-300:      3px !default;
+$ouds-elevation-blur-400:      4px !default;
+// $ouds-elevation-blur-500:      8px !default;
+$ouds-elevation-blur-600:      12px !default;
+// $ouds-elevation-blur-700:      20px !default;
+
+// $ouds-elevation-spread-n400:   -8px !default;
+$ouds-elevation-spread-n300:   -4px !default;
+// $ouds-elevation-spread-n200:   -2px !default;
+$ouds-elevation-spread-n100:   -1px !default;
+$ouds-elevation-spread-0:      0 !default;
+$ouds-elevation-spread-300:    3px !default;
+// scss-docs-end ouds-raw-shadow
```

Comment:
Not 100% sure yet, but we could use composite tokens too (if technically possible), something like:
`$elevation-bottom-0: $ouds-elevation-x-none $ouds-elevation-y-none $ouds-elevation-blur-none $ouds-elevation-spread-none $ouds-elevation-color-none` to define `$elevation-none: $elevation-bottom-0` 🤷